settings/Search.setting.php

```diff
@@ -228,4 +228,24 @@
     'description' => ts("Which fields can be searched on in the menubar quicksearch box? Don't see your custom field here? Make sure it is marked as Searchable."),
     'help_text' => NULL,
   ],
+  'default_pager_size' => [
+    'group_name' => 'Search Preferences',
+    'group' => 'Search Preferences',
+    'name' => 'default_pager_size',
+    'type' => 'Integer',
+    'quick_form_type' => 'Element',
+    'html_type' => 'text',
+    'html_attributes' => [
+      'size' => 2,
+      'maxlength' => 3,
+    ],
+    'default' => 50,
+    'add' => '5.33',
+    'title' => ts('Default Search Pager size'),
+    'is_domain' => 1,
+    'is_contact' => 0,
+    'description' => ts('What is the default number of records to show on a search'),
+    'help_text' => ts('Set this to show a specific number of records'),
```

Comment:
There's an open ticket about this somewhere that help_text doesn't work in the "new" settings system and it causes an error: `User warning: Smarty error: unable to read resource: "CRM\Admin\Form\Setting\Search.hlp" in Smarty->trigger_error() (line 1108 of ...\packages\Smarty\Smarty.class.php)`. Until that's fixed you'll need to either leave out the help_text here or put some smarty in the tpl. The description here seems adequate so I think can leave it out.

Reply:
ok removed